tests/test_level/test_level_02.py

```diff
@@ -654,6 +673,12 @@ def test_assign(self):
 
         self.multi_level_tester(code=code, expected=expected, max_level=11, unused_allowed=True)
 
+    def test_assign_micro_bit(self):
+        code = "naam is Felienne"
+        expected = "    naam = 'Felienne'"
```

Comment:
The test fails locally. I think there shouldn't be any leading whitespace. Please make sure that all tests are passing and when you are ready, convert the PR to ready for review. Then, I think we can move to the other PR

Reply:
Yes indeed i ran only the new tests. After i ran all tests locally there was one more failing test as well. Fixed that but will check few things before i convert it for review.

Reply:
Everything should be fine now until level 4